modules/core/src/scenegraph/gltf/gltf-material-parser.js

```diff
@@ -93,7 +23,7 @@ export default class GLTFMaterialParser {
     this.parameters = {};
 
     if (ibl) {
-      this.env = new GLTFEnv(gl);
+      this.env = ibl;
```

Comment:
I feel that `ibl` is too terse, we need a better name. Type out whatever it means or use another name.

Reply:
`Image Based Rendering or IBL : a technique that uses an image as a light source`

Reply:
I replaced all of references to `imageBasedLighting`. Should I also rename environment?

Reply:
> I replaced all of references to imageBasedLighting.

OK thanks, let's go with that for now. 

> Should I also rename environment?

Yes, maybe `ImageBasedLightingEnvironment`, or `IBLEnvironment`.

I think writing a doc about that class, with short explanation, references to papers, references to the tool you mention below would be really helpful.

I assume the PBR module in some sense depends on or "collaborates with" this class, making that relationship crips and clear in the docs (perhaps by updating PBR docs?) would surely cast this PR in a much clearer light.